packages/keyring-snap-sdk/src/methods.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  array,
+  literal,
+  object,
+  string,
+  type Infer,
+} from '@metamask/superstruct';
+
+export enum KeyringMethod {
```

Comment:
Actually, we should name this `SnapManageAccountsMethod`

Since it's meant to be used with `snap_manageAccounts`, this makes more sense IMO

Reply:
Ok cool, makes sense.